jdbc/src/java/org/apache/hive/jdbc/HttpRequestInterceptorBase.java

```diff
@@ -99,7 +118,20 @@ public void process(HttpRequest httpRequest, HttpContext httpContext)
         httpRequest.addHeader("Cookie", cookieHeaderKeyValues);
       }
     } catch (Exception e) {
-      throw new HttpException(e.getMessage(), e);
+      throw new RuntimeException(e);
```

Comment:
What is the reason for changing the exception type?

Reply:
oh, this is a leftover, I'll remove